scheduler/replica_scheduler.go

```diff
@@ -45,13 +45,27 @@ func NewReplicaScheduler(ds *datastore.DataStore) *ReplicaScheduler {
 	return rcScheduler
 }
 
+func isFirstAttachment(rs map[string]*longhorn.Replica) bool {
```

Comment:
Why not directly check the Node ID or DIsk ID...?

Reply:
Do you mean to check `replica.Spec.NodeID`?

Reply:
Yes

Reply:
Yeah, it could check NodeId. Thanks.
Modified.